From d87469e00e9b60cf36bec5295847da066f65fae9 Mon Sep 17 00:00:00 2001
From: WGH <wgh@torlan.ru>
Date: Mon, 17 Dec 2018 20:49:31 +0300
Subject: [PATCH 2/8] Specify Python version explicitly in shebangs

Fixes compilation failures in distributions that have 'python' refer
to Python 3.

See issue client9/libinjection#92.

That was originally posted at: client9/libinjection#144
---
 src/fingerprints2sqli.py | 2 +-
 src/make_parens.py       | 2 +-
 src/sqlparse2c.py        | 2 +-
 src/sqlparse_map.py      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/fingerprints2sqli.py b/src/fingerprints2sqli.py
index 4fe1890..0c80ed1 100755
--- a/src/fingerprints2sqli.py
+++ b/src/fingerprints2sqli.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 """
 Small script to convert fingerprints back to SQL or SQLi
 """
diff --git a/src/make_parens.py b/src/make_parens.py
index 554a08c..8d610b7 100755
--- a/src/make_parens.py
+++ b/src/make_parens.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # pylint: disable=C0103,R0911,R0912,R0915
 # disable short-variable-names, too many branches, returns, statements
 """
diff --git a/src/sqlparse2c.py b/src/sqlparse2c.py
index 5b37a5b..d705dbe 100755
--- a/src/sqlparse2c.py
+++ b/src/sqlparse2c.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 #  Copyright 2012, 2013 Nick Galbreath
 #  nickg@client9.com
diff --git a/src/sqlparse_map.py b/src/sqlparse_map.py
index 7c2797c..67aa040 100755
--- a/src/sqlparse_map.py
+++ b/src/sqlparse_map.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # pylint: disable=C0301,C0302
 # Turn off line-too-long, and too-many-lines warnings
 #
-- 
2.27.0

